common/channel.h

```diff
@@ -159,9 +159,9 @@ absl::StatusOr<SystemControlBlock *>
 CreateSystemControlBlock(toolbelt::FileDescriptor &fd);
 
 struct SlotBuffer {
-  SlotBuffer(int32_t slot_size) : slot_size(slot_size) {}
-  SlotBuffer(int32_t slot_size, toolbelt::FileDescriptor fd)
-      : slot_size(slot_size), fd(std::move(fd)) {}
+  SlotBuffer(int32_t slot_size_) : slot_size(slot_size_) {}
```

Comment:
Here too

Reply:
Same, `-Wshadow-field-in-constructor` warnings.